Tests/feaLib/builder_test.py

```diff
@@ -618,7 +628,6 @@ def test_pairPos_enumRuleOverridenBySinglePair_DEBUG(self):
                 "} test;")
         captor.assertRegex('Already defined position for pair A V at')
 
-
```

Comment:
don't remove this line here. between definitions at module-level there needs to be two line breaks, as per pep8 and black